pkg/loki/modules.go

```diff
@@ -382,6 +384,29 @@ func (t *Loki) initDistributor() (services.Service, error) {
 	return t.distributor, nil
 }
 
+func (t *Loki) initIngestLimits() (services.Service, error) {
+	if !t.Cfg.KafkaConfig.IngestLimits.Enabled {
```

Comment:
Hmm, it seems not, we have a second guard here?

Reply:
> Answering here for the above. The guards above are obsolete and predate the `initIngestLimits`, so keeping this one here and remove the above.